incubator/hnc/internal/kubectl/tree.go

```diff
@@ -42,6 +43,7 @@ var treeCmd = &cobra.Command{
 		flags := cmd.Flags()
 		footnotesByMsg = map[string]int{}
 		footnotes = []string{}
+		hasSubnamespace = false
```

Comment:
This isn't needed - boolean variables default to `false`.

Reply:
done